dhall/src/Dhall.hs

```diff
@@ -1031,6 +1031,10 @@ data InterpretOptions = InterpretOptions
     , constructorModifier :: Text -> Text
     -- ^ Function used to transform Haskell constructor names into their
     --   corresponding Dhall alternative names
+    , collapseSingletonRecords :: Bool
+    -- ^ Set this to `True` if you want the corresponding Dhall type to collapse
+    --   records with one field by replacing the record with the underlying
+    --   field
```

Comment:
Understanding this description requires knowing which types map to Dhall records. Examples would probably make this clearer.

Reply:
I think it would be nice if the option would differentiate between named and anonymous fields in Haskell. IMHO this option makes sense for named fields, but not so much for anonymous ones.